README.md

```diff
@@ -472,7 +474,7 @@ limit on private IPs allowed by your instance. Setting or resetting of `ENABLE_P
 
 ---
 
-`WARM_PREFIX_TARGET` 
+#### `WARM_PREFIX_TARGET` (Since v1.9)
```

Comment:
nit: How about changing it to "v1.9+" instead?

Reply:
Yeah, I can do that.